Assets/MixedRealityToolkit/_Core/Extensions/CollectionsExtensions.cs

```diff
@@ -150,12 +150,35 @@ public static T[] ExportDictionaryValuesAsArray<T>(this Dictionary<uint, T> inpu
         /// <param name="input">The InteractionDefinition dictionary reference (generics, performed on a Dictionary)</param>
         /// <param name="key">The specific DeviceInputType value to update</param>
         public static bool GetDictionaryValueChanged(
-            this Dictionary<Definitions.Devices.DeviceInputType, IMixedRealityInteractionMapping> input, Definitions.Devices.DeviceInputType key)
+            this Dictionary<Definitions.Devices.DeviceInputType, MixedRealityInteractionMapping> input, DeviceInputType key)
         {
             var entry = input[key];
             var changed = entry.Changed;
             input[key] = entry;
             return changed;
         }
+
+
+        /// <summary>
+
+        /// Overload extension to enable getting of an InteractionDefinition of a specific type
+
+        /// </summary>
+
+        /// <param name="input">The InteractionDefinition array reference</param>
+
+        /// <param name="key">The specific DeviceInputType value to query</param>
+
```

Comment:
Formatting is strange. Extra spaces in each line

Reply:
Possibly how I copied it, do you just want to fix that afterwards?